app/pages/project/stats/stats.cjsx

```diff
@@ -178,14 +178,14 @@ ProjectStatsPage = React.createClass
         classificationFootnote =
           <span className="project-stats-footer">
            {classificationFootnoteMarker}
-            The gap in the classification data from {moment(classificationGap[0]).format 'MMM-DD-YYYY'} to {moment(classificationGap[1]).format 'MMM-DD-YYYY'} was caused a bug in our event notification system.  <b>No</b> classifications were lost in this time. 
+            Due to an issue with our stats server all data before {moment(classificationGap[1]).format 'MMM-DD-YYYY'} is currently unavailable.  We are currently working to resolve this issue.  <b>No</b> classifications were lost in this time. 
```

Comment:
not sure if date referenced in explanation should be index `0` or `1` based on line #167 comment?